src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -392,22 +393,24 @@ public abstract class TreeOptions : TrainerInputBaseWithGroupId
         // REVIEW: Not used.
```

Comment:
>/ REVIEW: Not used. [](start = 9, length = 19)

Is it true?

Reply:
True. Will remove it.

---
In reply to: [261814752](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261814752) [](ancestors = 261814752)